### ISSUE TITLE
Does Brigadier support all mac models?

### ISSUE THREAD

**Issue by PrimalGroudo** [timsutton/brigadier#106]
My mac model is Mac14,15 and brigadier can't find it so i was just assuming that mine isn't supported

**Comment by ExpiredCoder** [comment]
There is not currently a brigadier version that supports this model. Sadly, brigadier is not really updated anymore so chances are, you won't find anything here. Sorry about that!

**Comment by m1ctk** [comment]
Yes the ./brigadier cmd does nothing with my 2012 mac book running sonoma. Is there any other way of installing windows 10 only ?

**Comment by fabd** [comment]
Fyi on my first try it failed because it's case sensitive : `MacbookPro5,5` -->  `MacBookPro5,5`
